app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsDiffDetailView.kt

```diff
@@ -0,0 +1,35 @@
+package org.wikipedia.suggestededits
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.LinearLayout
+import androidx.annotation.DrawableRes
+import kotlinx.android.synthetic.main.view_suggested_edits_contribution_diff_detail.view.*
+import org.wikipedia.R
+
+class SuggestedEditsDiffDetailView @JvmOverloads constructor(
+        context: Context,
+        attrs: AttributeSet? = null,
+        defStyle: Int = 0,
+        defStyleRes: Int = 0
+) : LinearLayout(context, attrs, defStyle, defStyleRes) {
```

Comment:
These can be simplified. (see the previous comments about removing `@JvmOverloads`